GANDLF/entrypoints/verify_install.py

```diff
@@ -0,0 +1,56 @@
+#!usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os, argparse, sys
+import click
+from deprecated import deprecated
+
+from GANDLF.entrypoints import append_copyright_to_help
+from GANDLF.utils import logger_setup
+
+
+def _verify_install():
+    try:
+        import GANDLF as gf
+
+        print("GaNDLF installed version:", gf.__version__)
+    except Exception as e:
+        raise Exception(
+            "GaNDLF not properly installed, please see https://mlcommons.github.io/GaNDLF/setup"
+        ) from e
+
+    # we always want to do submodule update to ensure any hash updates are ingested correctly
+    try:
```

Comment:
Two questions - first I do not see anything git related here anymore, so perhaps comment is outdated. Secondly, conceptual question - should we indeed try to install the Gandlf for user if the version check fails? Shouldn't we just throw an error and ask to manually do it? Automated install here can go in some undesired ways (although as of right now cannot think of any). Thing to discuss, perhaps I am paranoid

Reply:
Agreed.